addOns/bruteforce/CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
 
+## [10] - 2020-03-04
+### Added
+- option and functionality to find files without extension
```

Comment:
Can you turn this into a sentence please. (Leading cap and period, maybe also add an issue reference `(Issue 5883).`